src/fastmcp/client/sampling/handlers/anthropic.py

```
@@ -0,0 +1,366 @@
+                continue
+
+            raise ValueError(f"Unsupported content type: {type(content)}")
+
+        return anthropic_messages
+
+    @staticmethod
+    def _message_to_create_message_result(
+        message: Message,
+    ) -> CreateMessageResult:
+        if len(message.content) == 0:
+            raise ValueError("No content in response from Anthropic")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)